### ISSUE TITLE
Running Flexirest with warnings enabled

### ISSUE THREAD

**Issue by lowjoel** [andyjeffries/flexirest#30]
This isn't _terribly_ important, but it would be nice to be able to keep warning counts down.

```
  Flexirest (0.7ms) RSpec::ExampleGroups::CoursemologyEvaluatorModelsBase::ModelKey::DummyModel#save
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/request.rb:245: warning: instance variable @forced_url not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/request_filtering.rb:24: warning: instance variable @before_filters not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/configuration.rb:85: warning: instance variable @request_body_type not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/configuration.rb:85: warning: instance variable @request_body_type not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/configuration.rb:85: warning: instance variable @request_body_type not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/configuration.rb:182: warning: instance variable @proxy not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/request.rb:245: warning: instance variable @forced_url not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/request_filtering.rb:24: warning: instance variable @before_filters not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/configuration.rb:85: warning: instance variable @request_body_type not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/configuration.rb:85: warning: instance variable @request_body_type not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/configuration.rb:85: warning: instance variable @request_body_type not initialized
vendor/bundle/ruby/2.3.0/gems/flexirest-1.2.16/lib/flexirest/configuration.rb:182: warning: instance variable @proxy not initialized
  Flexirest (0.8ms) RSpec::ExampleGroups::CoursemologyEvaluatorModelsBase::ModelKey::DummyModel#save

```

**Comment by lowjoel** [comment]
Wow, thanks @andyjeffries I'll give it a shot.

**Comment by lowjoel** [comment]
That helped a lot, but there's still more... https://travis-ci.org/Coursemology/evaluator-slave/jobs/120010082

**Comment by andyjeffries** [comment]
No need, just fixed those ones in v1.3.7

**Comment by lowjoel** [comment]
@andyjeffries sweet!
